### ISSUE TITLE
Helper functions do not return notification object

### ISSUE THREAD

**Issue by IceDragon200** [mansona/ember-cli-notifications#121]
The following code will quickly explain the problem:

```javascript
// somewhere in ember code, where magic happens
const notify = this.get("notifications"); // for convience sake
const heldNote = notify.info("Be patient, we're doing things");

//... do some stuff that will take awhile

notify.removeNotification(heldNote);
notify.success("And we're done!");
```

Sadly the [helper methods](https://github.com/stonecircle/ember-cli-notifications/blob/master/addon/services/notification-messages-service.js#L37-L64) do not return the notification object, making this impossible, without resorting to `addNotification` directly, or overwriting them in a extended service.

It would also be nice if the functions there would return something useful (helpers should return the `notification` for aforementioned reason), while others should return the notification service for function chaining.

```javascript
notify.clearAll().setDefaultAutoClear(true).setDefaultClearNotification(5000);
```

Also, shouldn't `setDefaultClearNotification` be `setDefaultClearDuration` ?

**Comment by ynnoj** [comment]
> Also, shouldn't `setDefaultClearNotification` be `setDefaultClearDuration` ?

☝️ #122

**Comment by ynnoj** [comment]
Fixed by #151.
